src/main/java/uk/gov/digital/ho/hocs/info/api/dto/WorkstackColumnDto.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.digital.ho.hocs.info.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.digital.ho.hocs.info.domain.model.WorkstackColumn;
+
+
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+public class WorkstackColumnDto {
+
+
+    @JsonProperty("displayName")
+    private String displayName;
+
+    @JsonProperty("dataAdapter")
+    private String dataAdapter;
+
+    @JsonProperty("renderer")
+    private String renderer;
+
+    @JsonProperty("dataValueKey")
+    private String dataValueKey;
+
+    @JsonProperty("isFilterable")
+    private boolean filterable;
+
+    @JsonProperty("headerClassName")
+    private String headerClassName;
+
+    public static WorkstackColumnDto from(WorkstackColumn column) {
```

Comment:
I think there should be a test file for this method

Reply:
Done